app/assets/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,18 @@
+.alert-container{
+  box-sizing: border-box;
+  position: absolute;
+  z-index: 1;
+  margin-left: 50px;
+  background-color: rgb(26, 26, 26);
+  width: 350px;
+  color: $gold;
+  text-align: center;
+  font-size: 22px;
+  border: 1px solid $gold;
+  border-radius: 15px;
+  padding: 10px;
+
+  span{
+    font-size: 20px;
+  }
+}
```

Comment:
The `box-sizing` is on by default for every element on the website because we set it up in the setup file so you can remove it.

Can you change the `z-index` to 10 instead please ? 

With position abolute it's better to use `left` and `top` propreties instead of `margin`. Can you remove the margin and add `left: 50px;` instead please ?

background-color should either be `$main-grey` or `$secondary-grey`